std/os/index.zig

```diff
@@ -39,6 +39,12 @@ pub const File = @import("file.zig").File;
 pub const time = @import("time.zig");
 
 pub const page_size = 4 * 1024;
```

Comment:
I understand this isn't your code, but should this be using [sysconf](http://man7.org/linux/man-pages/man3/sysconf.3.html)?

Reply:
I believe you can determine the page size based on the tuple of (operating system, architecture) which importantly means that  `page_size` can be a comptime-known value. Currently all our supported targets have the same page size. This value computation will get more complicated as we add more supported targets to the standard library.